packages/build/src/core/constants.js

```diff
@@ -82,12 +82,12 @@ const getEdgeHandlersSrc = async function (edgeHandlers, buildDir) {
     return edgeHandlers
   }
 
-  if (await isDirectory(`${buildDir}/${DEFAULT_EDGE_HANDLERS_SRC}`)) {
-    return DEFAULT_EDGE_HANDLERS_SRC
+  if (await isDirectory(`${buildDir}/${DEFAULT_HANDLERS_SRC}`)) {
```

Comment:
The longer name is more explicit here and I think it is better, WDYT?

Reply:
:+1: